doc/How_To_Install_JDK.md

```diff
@@ -1,19 +1,12 @@
 How to Install Java Development Kit (JDK)
```

Comment:
Considering the "Contributor Setup Instructions For MapTool" contains the actual instructions, and it is already referenced in `Contributing.md`, would it make sense to remove this file?

Reply:
Removed